packages/lib-classifier/src/plugins/tasks/components/TaskInput/TaskInput.spec.js

```diff
@@ -11,54 +14,69 @@ const radioTypeAnnotation = {
 }
 
 describe('TaskInput', function () {
-  describe('render', function () {
-    let wrapper
-    before(function () {
-      wrapper = shallow(<TaskInput annotation={radioTypeAnnotation} index={0} type='radio' />)
-    })
-
-    it('should render without crashing', function () {
-      expect(wrapper).to.be.ok()
-    })
+  function withGrommet() {
+    return function Wrapper({ children }) {
+      return (
+        <Grommet theme={zooTheme}>
+          {children}
+        </Grommet>
+      )
+    }
+  }
 
-    it('should render a StyledTaskInput', function () {
-      expect(wrapper.find(StyledTaskInput)).to.have.lengthOf(1)
-    })
-
-    it('should render a StyledTaskLabel', function () {
-      expect(wrapper.find(StyledTaskLabel)).to.have.lengthOf(1)
+  describe('render', function () {
+    it('should render a radio button', function () {
+      render(
+        <TaskInput annotation={radioTypeAnnotation} index={0} type='radio' />,
```

Comment:
The `annotation=radioTypeAnnotation` doesn't do anything in this spec file.  I'd move it to a comment for context or remove it.

Reply:
Oh yeah. There isn't an `annotation` prop in `TaskInput`. I wonder why this was in the code.